dev_settings.py

```diff
@@ -82,12 +85,20 @@
         'COUCH_HTTPS': False,
         'COUCH_SERVER_ROOT': 'localhost:5984',
         'COUCH_USERNAME': 'commcarehq',
-        'COUCH_PASSWORD': 'commcarehq',
+        'COUCH_PASSWORD': '',
```

Comment:
my localsettings have the username blank and the password as `commcarehq` - but that could be due to something I've done locally.

Reply:
tested locally and you can clear either or both. i just removed the username also. https://github.com/dimagi/commcare-hq/pull/29625/commits/3e65f1d6b0ab030304d999b0f3921d46361b2afe